frontend/src/pages/learn/planning-to-save-for-retirement.tsx

```diff
@@ -58,65 +59,55 @@ const PlanningToSaveForRetirement: FC = () => {
-            <p>{t('turning-savings-into-income-content-one')}</p>
-            <p>{t('turning-savings-into-income-content-two')}</p>
-            <Image src={`/assets/rrif_${router.locale}.svg`} height={500} width={500} alt={t('rrif-alt')} className='mx-auto'/>
-            <p>{t('turning-savings-into-income-content-three')}</p>
-            <Trans i18nKey='turning-savings-into-income-content-four' components={{anchor: <a href={t("RRIF-link")} target='_blank' rel="noopener noreferrer" className='no-after'/>}}>
-              <p>{t('turning-savings-into-income-content-four')}</p>
-            </Trans>
-          </div>
-          <div className='space-y-10'>
-            <h2 className='h2'>{t('learn-more-heading')}</h2>
-            <a href='#' tabIndex={0} className='flex space-between m-5 items-center no-underline text-black visited:text-black border-b-2'>
-              <div className='md:mr-20'>
-                <h3 className='font-medium mb-2'>{t('transitioning-heading')}</h3>
-                <p className='text-[.9em] text-gray-700'>{t('transitioning-content')}</p>
-              </div>
-              <MdArrowForwardIos className='font-bold ml-5 text-[5rem] xl:text-2xl md:mr-5' />
-            </a>
-            <a href='https://www.canada.ca/en/services/benefits/publicpensions/cpp.html' tabIndex={0} className='flex space-between m-5 items-center no-underline text-black visited:text-black border-b-2'>
-              <div className='md:mr-20'>
-                <h3 className='font-medium mb-2'>{t('cpp-heading')}</h3>
-                <p className='text-[.9em] text-gray-700'>{t('cpp-content')}</p>
-              </div>
-              <MdArrowForwardIos className='font-bold ml-5 text-[5rem] xl:text-2xl md:mr-5' />
-            </a>
-            <a href='https://www.canada.ca/en/services/benefits/publicpensions/cpp/old-age-security.html' tabIndex={0} className='flex space-between m-5 items-center no-underline text-black visited:text-black border-b-2'>
-              <div className='md:mr-20'>
-                <h3 className='font-medium mb-2'>{t('oas-heading')}</h3>
-                <p className='text-[.9em] text-gray-700'>{t('oas-content')}</p>
-              </div>
-              <MdArrowForwardIos className='font-bold ml-5 text-[5rem] xl:text-2xl md:mr-5' />
-            </a>
-            <a href='#' tabIndex={0} className='flex space-between m-5 items-center no-underline text-black visited:text-black border-b-2'>
-              <div className='md:mr-20'>
-                <h3 className='font-medium mb-2'>{t('sources-of-income-heading')}</h3>
-                <p className='text-[.9em] text-gray-700'>{t('sources-of-income-content')}</p>
-              </div>
-              <MdArrowForwardIos className='font-bold ml-5 text-[5rem] xl:text-2xl md:mr-5' />
-            </a>
-          </div>
+          <h2 id="overview" className='invisible h-0'>{t('overview-link-text')}</h2>
```

Comment:
I think the class should be `sr-only` instead of `invisible h-0`.
https://tailwindcss.com/docs/screen-readers